### ISSUE TITLE
(R3-031) - (MED) - MED BUN and REP BUN issues

### ISSUE THREAD

**Issue by anthonyperk** [CurrentlyNotSet/SERB#599]
Per your request for a new ticket on 7/18/17 (Original ticket R3-021)

After talking with Staff at SERB… we would like to make sure that the MED section BUN number cannot
be populated if it does not match something from the REP Section BUN table. Can you add that to your
notes on this ticket. This is due to the issue of BUN numbers not existing in the REP section for
grandfathered employers who have not had a REP case.

-------------------------------------------------------------------------------------------------------------

Mary is updating the case information within the RED box for the following cases…
2017-MED-07-0818, 2017-MED-07-0819, 2017-MED-07-0820
When she is switching from case 0818 to 0819 the page refreshes with updated info… When she switches
from 0819 to 0820 the info does not refresh. When I switch from 0820 back to 0818 The info does not
refresh. If I close out and reopen SMDS it works when I go to case 0818. If I close out and go to case 0820
this info is blank.
I am hoping you can run this through your debugger to locate what the glitch is.

![image](https://user-images.githubusercontent.com/24595073/29043165-f35e922a-7b87-11e7-8460-64dcff83cd91.png)

![image](https://user-images.githubusercontent.com/24595073/29043169-f68021ee-7b87-11e7-88a3-eb2b08d00e0a.png)

![image](https://user-images.githubusercontent.com/24595073/29043174-fcd967b2-7b87-11e7-894d-86d8d85fe303.png)

**Comment by parkjohnston** [comment]
Please follow up with the follow question: 

Would we be checking this on selection from the search or when they would manually enter the value? 

This answer will drive the complexity! 

Thanks!

**Comment by parkjohnston** [comment]
Never got an answer on this....back to waiting on answer tag

**Comment by parkjohnston** [comment]
to satisfy this request...choice has been made to only allow searching for the number. This is 110% make sure that the BUN number is valid which he asked for

**Comment by parkjohnston** [comment]
Have updated this as requested, new functionality has been implemented. 

Double Click -- Allows for search based on the REP table as requested.
Right Click (Double) -- Allows for data to be cleared as with the rest of the application.

**Comment by parkjohnston** [comment]
Time 1.5 hour
